run.sh

```diff
@@ -91,6 +108,8 @@ args=(
   --rpc-drone-address 127.0.0.1:9900
   --accounts "$dataDir"/accounts
   --log -
+  --enable-rpc-exit
```

Comment:
Specifically needed for `run-sanity.sh`; Also, I think this is safe in the security aspect; after all, `run.sh` is a developer helper script.